scripts/irods/test/test_pam_password_authentication.py

```diff
@@ -176,91 +202,70 @@ def test_password_max_time_less_than_password_min_time_makes_ttl_constraints_uns
         original_max_time = self.admin.assert_icommand(
             ['iadmin', 'get_grid_configuration', self.configuration_namespace, max_time_option_name], 'STDOUT')[1].strip()
 
-        IrodsController().stop()
-
         auth_session_env_backup = copy.deepcopy(self.auth_session.environment_file_contents)
-        admin_session_env_backup = copy.deepcopy(self.admin.environment_file_contents)
         try:
-            service_account_environment_file_path = os.path.join(
-                os.path.expanduser('~'), '.irods', 'irods_environment.json')
-            with lib.file_backed_up(service_account_environment_file_path):
-                client_update = test_configurations.make_dict_for_ssl_client_environment(
-                    self.server_key_path, self.chain_pem_path, self.dhparams_pem_path)
+            client_update = {'irods_authentication_scheme': self.authentication_scheme}
+            self.auth_session.environment_file_contents.update(client_update)
 
-                lib.update_json_file_from_dict(service_account_environment_file_path, client_update)
+            # Make sure the settings applied correctly...
+            self.admin.assert_icommand(
+                ['iadmin', 'get_grid_configuration', self.configuration_namespace, max_time_option_name],
+                'STDOUT', original_max_time)
```

Comment:
This assertion seems a bit strange. I don't see where the value is being changed? I haven't spotted how this can fail.

Reply:
Removed